### PR TITLE
Feature/68 unit testing

### PR DESCRIPTION
**Pull request by HakobTono**

### CODE REVIEW

File: src/test/java/com/epam/emotionalhelp/service/impl/EmotionServiceImplTest.java [hunk 1]
```diff
@@ -117,19 +135,33 @@ void delete_withCorrectEmotionId() {
...
+
+        Emotion createEmotion() {
+            return new Emotion("emotion");
+        }
+
+        Emotion createAnotherEmotion() {
+            return new Emotion("emotion2");
+        }
+
+
```
Comment (maximelyashevich) [inline src/test/java/com/epam/emotionalhelp/service/impl/EmotionServiceImplTest.java:156]:
```suggestion
        EmotionRequestDto createEmotionDTO() {
            var emotionDto = new EmotionRequestDto();
            emotionDto.setDescription(EMOTION_VALUE);
            return emotionDto;
        }

        EmotionRequestDto createAnotherEmotionDTO() {
            var emotionDto = new EmotionRequestDto();
            emotionDto.setDescription("emotion1");
            return emotionDto;
        }
        
```

File: src/test/java/com/epam/emotionalhelp/service/impl/EmotionServiceImplTest.java [hunk 2]
```diff
@@ -117,19 +135,33 @@ void delete_withCorrectEmotionId() {
...
+        List<Emotion> generateEmotionList() {
+            List<Emotion> emotions = new ArrayList<>();
+            emotions.add(new Emotion("emotion"));
+            emotions.add(new Emotion("emotion1"));
+            emotions.add(new Emotion("emotion2"));
+            emotions.add(new Emotion("emotion3"));
+            emotions.add(new Emotion("emotion4"));
+            return emotions;
+        }
```
Comment (maximelyashevich) [inline src/test/java/com/epam/emotionalhelp/service/impl/EmotionServiceImplTest.java:165]:
```suggestion
        List<Emotion> generateEmotionList() {
            return List.of(
                    new Emotion("emotion"),
                    new Emotion("emotion1"),
                    new Emotion("emotion2"),
                    new Emotion("emotion3"),
                    new Emotion("emotion4")
            );
        }
```

**Review by maximelyashevich — CHANGES_REQUESTED** [review]
Hi @HakobTono,
Please refactor your all unit tests based on my recommendation for `EmotionServiceImplTest` class.

Regards,
Maks

**Comment by kolesroma** [comment]
Hi, @HakobTono 
why build failed? Try to make changes so classes compile

**Review by kolesroma — CHANGES_REQUESTED** [review]
.

**Pull request merged**